docs/source/user-guide/cli.md

```diff
@@ -388,7 +411,7 @@ Note that the `endpoint` format of oss needs to be: `https://{bucket}.{oss-regio
 CREATE EXTERNAL TABLE test
 STORED AS PARQUET
 OPTIONS(
-    'service_account_path' '/tmp/gcs.json',
+    'gcp.service_account_path' '/tmp/gcs.json',
```

Comment:
is it related to cos onboard?

Reply:
No, add the missing prefix gcp and aws to related external table options.